cmd/common/utils.go

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
IMO, this `utils.go` should be moved inside commonco package.

Reply:
Changed it